shuup/front/admin_module/carts/views/_detail.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2020, Shoop Commerce Ltd. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.views.generic import DetailView
+
+from shuup.admin.supplier_provider import get_supplier
+from shuup.core.basket.objects import BaseBasket
+from shuup.core.models import OrderLineType
+from shuup.front.models import StoredBasket
+
+
+class CartDetailView(DetailView):
+    model = StoredBasket
+    template_name = 'shuup/front/admin/stored_basket_detail.jinja'
+
+    def get_queryset(self):
+        qs = super(CartDetailView, self).get_queryset()
+        return qs.prefetch_related("products")
+
+    def get_context_data(self, **kwargs):
+        context = super(CartDetailView, self).get_context_data(**kwargs)
+
+        basket = BaseBasket(
```

Comment:
Not sure this is necessary. But this makes it easy to render the template. Also some thought here would be to maybe option to pass this for order creator.